packages/shared/src/common/retry.ts

```diff
@@ -107,7 +108,13 @@ export async function executeWithRetries<T>(
         throw error;
       }
       log(`${msg}, retrying... (attempt: ${attempt})`);
-      const timeToWait = getTimeToWait({ initialDelay, backoffMultiplier, attempt, maxDelay });
+      const timeToWait = getTimeToWait({
+        initialDelay,
+        backoffMultiplier,
+        attempt,
+        maxDelay,
+        error,
```

Comment:
Same comment as below -- adding error but not seeming to change any code to do anything with it.

Reply:
responded there but the initialDelay is dependent on the error